personhood/db.go

```diff
@@ -48,15 +48,22 @@ func (s *Service) tryLoad() error {
 }
 
 type storage1 struct {
```

Comment:
because _storage_ is not available ?

Reply:
For a new version of the db, it will be called `storage2` and will evolve from `storage1`.